app/cdap/components/ConnectionList/index.tsx

```diff
@@ -234,39 +237,49 @@ export default function ConnectionList() {
                     variant="body2"
                     className={classes.headerLabel}
                     ref={(element) => {
-                      refs.current[index] = element;
+                      refs.current[tabIndex] = element;
                     }}
                   >
-                    {dataForTabs[index - 1].selectedTab}
+                    {dataForTabs[tabIndex - 1].selectedTab}
                   </Typography>
                 </Box>
               );
           }
           return (
-            <Box className={classes.tabsContainerWithHeader}>
+            <Box
+              className={classes.tabsContainerWithHeader}
+              data-testid={`connection-tabs-column-${tabIndex}`}
+            >
               <Box className={classes.tabHeaders}>{headerContent}</Box>
               <ConnectionsTabs
-                tabsData={each}
+                tabsData={eachTabItem}
                 handleChange={selectedTabValueHandler}
-                value={each.selectedTab}
-                index={index}
+                value={eachTabItem.selectedTab}
+                connectionColumnIndex={tabIndex}
                 connectionId={connectionId || ''}
                 toggleLoader={(value: boolean, isError?: boolean) => toggleLoader(value, isError)}
-                setIsErrorOnNoWorkSpace={setIsErrorOnNoWorkSpace}
+                setToaster={setToaster}
```

Comment:
again this should either be snackbar throughout or toaster throughout - please don't change the name of it

Reply:
Done